lib.xml

```diff
@@ -0,0 +1,4 @@
+<Ui xmlns="http://www.blizzard.com/wow/ui/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.blizzard.com/wow/ui/
+..\FrameXML\UI.xsd">
+	<Script file="LibSerialize.lua" />
```

Comment:
Please convert to spaces and add a trailing newline.

Reply:
Done. Note there is a trailing newline, though the github preview doesn't show that very well.

When there isn't a newline, you'll see this:
![image](https://user-images.githubusercontent.com/24878501/84960807-3eb1aa00-b0c8-11ea-9ba9-f1ccaf3631cb.png)